weave/trace/refs.py

```diff
@@ -327,7 +325,7 @@ def parse_uri(uri: str) -> CallRef:
 class DeletedRef(Ref):
     ref: Ref
     deleted_at: datetime
-    error: ObjectDeletedError
+    error: httpx.HTTPStatusError
```

Comment:
similar to the comment below, this should not be an http error, but a custom error. honestly, sort of odd for the deletedref to have an error anyway